### DISCUSSION TITLE
绳网小贴士01--不小心穿越了怎么办

### DISCUSSION THREAD

**Discussion by Ziew-Ryan** [share121/inter-knot#3004] [其他]
让我们从观察周遭环境吧！（目前仅收录“提瓦特”与“崩坏：星穹铁道”，欢迎绳匠踊跃补充）
1.观察世界

 1.1向居民询问“什么是元素”，观察回答，如果居民惊讶于你缺乏常识，那么可以确定，你进入的是名为“提瓦特”的世界，若居民同样表示 
 不清楚何为元素那么可以确定你大概率进入的是名为“崩坏：星穹铁道”的世界

2.分辨城市--“提瓦特”

 2.1通过建筑分辨所属城市
  2.1.1若建筑多为石头垒起，且可观测到一座石制站立姿态雕像，那么可确定你位于“蒙德”
  2.1.2若建筑多为木质结构，且拥有发黄的数目，那么可确定你位于“璃月”
  2.1.3若建筑多位木质结构，且可观测到一座石制跪姿雕像，那么可确定你位于”稻妻“
  2.1.4若城市依树而建，那么可确定你位于”须弥“
  2.1.5若城中有人形机械在巡逻，那么可确定你位于”枫丹“

 2.2通过政府名称分辨所属城市
  2.2.1西风骑士团--”蒙德“
  2.2.2天衡司--”璃月“
  2.2.3”三奉行“--”稻妻“
  2.2.4教令院--”须弥“
  2.2.5沫芒宫--”枫丹“

3.分辨城市--“崩坏：星穹铁道”（仅收录部分城市）

 3.1建筑风格
  3.1.1未来科技风，空间站--”黑塔“空间站
  3.1.2城内均为石板路，有有轨电车运行--雅利洛-VI
  3.1.3科幻里夹杂着一些中国风--仙舟联盟
  3.1.4华丽，城市就是大酒店--匹诺康尼

                                                                                                                                                                                         更新时间：2024/8/19

**Comment by Samu1KU** [comment]
非常有用，感谢！！
